tensorflow/compiler/xla/service/tuple_simplifier_test.cc

```diff
@@ -211,5 +217,76 @@ TEST_F(TupleSimplifierTest, IncompatibleTuples) {
   EXPECT_THAT(computation->root_instruction(), tuple);
 }
 
+TEST_F(TupleSimplifierTest, CanExcludeEntryComputation) {
+  //  Verify that the root computation can be excluded
```

Comment:
Nit, move this comment up one line?

Reply:
the other tests in the file seem to have their comments following the test declaration.  i think it is best to leave it here.